File: Front/Loja-App/models/Funcionario.ts
import { Loja } from "./Loja";
import { Telefone } from "./Telefone";

export interface Funcionario {
  id: number;
  nome: string;
  cpf: string;
  email: string;
  lojaId: number;
  loja: Loja;
  telefones: Telefone;
}

File: Back/src/ProjetoLoja.Domain/Funcionario.cs
using System.Collections.Generic;

namespace ProjetoLoja.Domain
{
    public class Funcionario
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string CPF { get; set; }
        public string Email { get; set; }
        public int LojaId { get; set; }
        public  Loja Loja { get; set; }
        public IEnumerable<Telefone> Telefones { get; set; }
    }
}